src/plugins/api_tests/src/core/external_script/base_scheme_interface/CInterfaceSasTests.cpp

```diff
@@ -31,12 +32,32 @@
 
 #include "SchemeSimilarityUtils.h"
 
-static const QString WORKING_DIR = U2::AppContext::getWorkingDirectoryPath();
-#ifndef Q_OS_MAC
-static const QString PROPER_WD_SCHEMES_PATH = WORKING_DIR + "/../../test/_common_data/cmdline/wd-sas-schemes/";
-#else
-static const QString PROPER_WD_SCHEMES_PATH = WORKING_DIR + "/../../../../../../test/_common_data/cmdline/wd-sas-schemes/";
-#endif
+#include <QFileInfo>
+
+static QString getTestDirImpl() {
+    QString testDir = qgetenv("UGENE_TESTS_PATH");
+    auto getDefaultTestDir = []() -> QString {
+        QString defaultTestDir;
+        #ifndef Q_OS_MAC
```

Comment:
nit: we have more readable isOsMac()/isOsLinux() methods.

Reply:
Fixed